conf/awsecr.config

```diff
@@ -0,0 +1,26 @@
+/*
+Config to set public AWS ECR images wherever possible
+This improves speed when running on AWS infrastructure.
+Use this as an example template when using your own private registry.
+*/
+
```

Comment:
Do we add a `TODO` statement to this config file as a reminder to update containers whenever they are updated in the pipeline. That's the thing that scares me the most by having this buried in a config.